src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.cs

```diff
@@ -2112,22 +2112,24 @@ private bool SkipComment()
                 }
                 else
                 {
-                    ThrowHelper.ThrowJsonReaderException(ref this, ExceptionResource.ExpectedStartOfValueNotFound, marker);
+                    ThrowHelper.ThrowJsonReaderException(ref this, ExceptionResource.UnexpectedEndOfDataWhileReadingComment);
                 }
             }
 
             if (IsLastSpan)
             {
-                ThrowHelper.ThrowJsonReaderException(ref this, ExceptionResource.ExpectedStartOfValueNotFound, JsonConstants.Slash);
+                ThrowHelper.ThrowJsonReaderException(ref this, ExceptionResource.UnexpectedEndOfDataWhileReadingComment);
```

Comment:
This should remain `ExpectedStartOfValueNotFound`

Reply:
Oh Didn't we want all the new exceptions that I had added to be removed?

Reply:
@ahsonkhan Please refer [here](https://github.com/dotnet/corefx/pull/35934#discussion_r284965929) and the following comment for the incorrectness of the original `ExpectedStartOfValueNotFound` exception.